### ISSUE TITLE
Request Money - Money Request button text is not center.

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#29759]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.3.85.0
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:**
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1697479665208989

## Action Performed:
1. Open any report.
2. Click <+> on the left of composer.
3. Manual input
4. Click next button

## Expected Result:
Button text must be align center.

## Actual Result:
Button text is not align center.

## Workaround:
Unknown 

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos
<details>
<summary>Android: Native</summary>

 
![Screenshot 2023-10-17 at 12 00 27 AM](https://github.com/Expensify/App/assets/93399543/7f4e1323-f545-4e8c-959c-ad23135d3707)

</details>

<details>
<summary>Android: mWeb Chrome</summary>

 
![Screenshot 2023-10-16 at 9 47 14 PM](https://github.com/Expensify/App/assets/93399543/f0b96442-63f5-4ecc-a4e8-7b7de05b3a09)

</details>

<details>
<summary>iOS: Native</summary>

 
![Screenshot 2023-10-16 at 10 04 57 PM](https://github.com/Expensify/App/assets/93399543/76e25605-2e60-4953-b153-2447396cbd9f)

</details>

<details>
<summary>iOS: mWeb Safari</summary>

 
![Screenshot 2023-10-16 at 9 43 08 PM](https://github.com/Expensify/App/assets/93399543/e6cb6269-5be8-4450-825d-5466119050cd)

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

 

https://github.com/Expensify/App/assets/93399543/5d69ab2d-5cbf-47fb-b4c0-3e67241a2914

![image (12)](https://github.com/Expensify/App/assets/93399543/bb45a758-bc68-4281-8fc2-b55943677f67)

</details>

<details>
<summary>MacOS: Desktop</summary>

 
![Screenshot 2023-10-16 at 9 53 15 PM](https://github.com/Expensify/App/assets/93399543/0108a1e4-bb17-4ef7-96fa-2a2590097936)

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by kbecciv** [comment]
## Proposal
### Please re-state the problem that we are trying to solve in this issue.
Money Request button text is not center.
### What is the root cause of that problem?
Button have padding: 8px 10px 8px 18px;
https://github.com/Expensify/App/blob/279ce1b6b0fbbfddea7735d90a120e934772da40/src/components/Button/index.js#L293
Paddings that define for button is not correct.
### What changes do you think we should make in order to solve the problem?
We need to make padding correctly on this
https://github.com/Expensify/App/blob/279ce1b6b0fbbfddea7735d90a120e934772da40/src/styles/styles.ts#L524
### What alternative solutions did you explore? (Optional)
Current button (edited)

**Comment by yh-0218** [comment]
## Proposal
### Please re-state the problem that we are trying to solve in this issue.
Money Request button text is not center.
### What is the root cause of that problem?
Button have padding: 8px 10px 8px 18px;
https://github.com/Expensify/App/blob/279ce1b6b0fbbfddea7735d90a120e934772da40/src/components/Button/index.js#L293
Paddings that define for button is not correct.
### What changes do you think we should make in order to solve the problem?
We need to make padding correctly on this
https://github.com/Expensify/App/blob/279ce1b6b0fbbfddea7735d90a120e934772da40/src/styles/styles.ts#L524
### What alternative solutions did you explore? (

**Comment by yh-0218** [comment]
Hi, @cristipaval PR is ready for review.

**Comment by yh-0218** [comment]
Hello, @narefyev91 On this step, what do I need to do?

**Comment by narefyev91** [comment]
> Hello, @narefyev91 On this step, what do I need to do?

Hey - nothing, we need Expensify engineer to approve and merge you PR - i already tag @cristipaval - if he will not be able - i will ping someone from Expensify

**Comment by Christinadobrzyn** [comment]
Since the PR is in review, I'm going to move this issue to weekly. Feel free to let me know if you'd prefer it remain daily.

**Comment by Pujan92** [comment]
Found a regression from this issue PR and reported [here](https://expensify.slack.com/archives/C049HHMV9SM/p1697713080450809)

**Comment by narefyev91** [comment]
I will say it's not a regression for this PR.
All Buttons were broken based on this PR which added not correct padding for all buttons https://github.com/Expensify/App/pull/12455/files
To fix issue which you linked @Pujan92 padding should be applied to icon and not to the whole button container

**Comment by yh-0218** [comment]
@Pujan92  @narefyev91 I will create New PR soon

**Comment by Pujan92** [comment]
I agree partially that the margin left needs to provide to an icon for correct alignment @narefyev91, but as we have updated the padding left in this PR I think that icon button case needs to be tested here only.

**Comment by Pujan92** [comment]
> @Pujan92 @narefyev91 I will create New PR soon

I think you need to wait for raising the PR untill that issue gets created by testing team.

**Comment by yh-0218** [comment]
Got it. @Pujan92

**Comment by yh-0218** [comment]
hi, @narefyev91 when can this task be completed?

**Comment by narefyev91** [comment]
@yh-0218 Hey! I think it should be close after 2 days from today

**Comment by Christinadobrzyn** [comment]
HI! I was assigned this payment [GH](https://github.com/Expensify/App/issues/29861) to pay @narefyev91 but I feel like it's a dupe of the payments needed for this job

@narefyev91 can you confirm if you should be paid [here](https://github.com/Expensify/App/issues/29861) and also in this GH?

Payment plan for review:

Issue Reporter: $50 @yh-0218
Contributor: $500 + $250 bonus @yh-0218
Contributor+: $500 + $250 bonus @narefyev91 

Eligible for 50% #urgency bonus? Y - @yh-0218 created [PR](https://github.com/Expensify/App/pull/29787) on October 17th merged on Oct 18th

Upwork job is [here](https://www.upwork.com/jobs/~0146a50c0ecd80150f). @yh-0218 and @narefyev91 can you apply to the job if the above breakdown makes sense?

**Comment by Christinadobrzyn** [comment]
I'm not sure why the Payment notification didn't happen for this [PR](https://github.com/Expensify/App/pull/29787) 

@cristipaval Is this ready to pay based on https://github.com/Expensify/App/issues/29759#issuecomment-1783396004

**Comment by narefyev91** [comment]
> HI! I was assigned this payment [GH](https://github.com/Expensify/App/issues/29861) to pay @narefyev91 but I feel like it's a dupe of the payments needed for this job
> 
> @narefyev91 can you confirm if you should be paid [here](https://github.com/Expensify/App/issues/29861) and also in this GH?
> 
> Payment plan for review:
> 
> Issue Reporter: $50 @yh-0218 Contributor: $500 + $250 bonus @yh-0218 Contributor+: $500 + $250 bonus @narefyev91
> 
> Eligible for 50% #urgency bonus? Y - @yh-0218 created [PR](https://github.com/Expensify/App/pull/29787) on October 17th merged on Oct 18th
> 
> Upwork job is [here](https://www.upwork.com/jobs/~0146a50c0ecd80150f). @yh-0218 and @narefyev91 can you apply to the job if the above breakdown makes sense?

Hey @Christinadobrzyn! I'm from expert agency Callstack - i think it should be included in a separate invoice. Thanks!

**Comment by cristipaval** [comment]
@Christinadobrzyn, the C+ here is from Callstack
The only payment due here is the reporting bonus and the contributor who fixed the issue (@yh-0218). No bonus applicable here, since there was a [regression](https://github.com/Expensify/App/issues/29967) after the fix

**Comment by Christinadobrzyn** [comment]
Thanks @cristipaval and @narefyev91  - updated the payment outline here - https://github.com/Expensify/App/issues/29759#issuecomment-1783396004

I'll keep this open and monitor for @yh-0218 to accept the Upwork offer!

**Comment by yh-0218** [comment]
hi, @Christinadobrzyn so my payment is $300?

**Comment by Christinadobrzyn** [comment]
hi @yh-0218 yes, that's correct! You'll be paid as the reporter and as the contributor.

**Comment by Christinadobrzyn** [comment]
I paid you in Upwork based on this payment structure - https://github.com/Expensify/App/issues/29759#issuecomment-1783396004

Let me know if you have any questions!
